lib/kaffe/group_member/worker/worker_supervisor.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Kaffe.WorkerSupervisor do
+  @moduledoc """
+  Supervise the individual workers.
+  """
+
+  use Supervisor
+
+  require Logger
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, :ok)
+  end
+
+  def start_worker_manager(pid) do
+    Supervisor.start_child(pid, worker(Kaffe.WorkerManager, []))
+  end
+
+  def start_worker(pid, message_handler, partition) do
+    Logger.debug "Starting worker for partition: #{partition}"
+    Supervisor.start_child(pid,
+      worker(Kaffe.Worker, [message_handler, partition], id: partition))
+  end
+
+  def init(:ok) do
+    Logger.info "event#startup=#{inspect self()}"
+
+    children = [
+    ]
```

Comment:
A `simple_one_for_one` is the same as a `one_for_one` except that it only knows about one kind of child, starts with zero, and can dynamically create children with a simple call. The children definition says if they should be permanent, transient, or temporary.